wit-manifest.json

```diff
@@ -235,7 +235,7 @@
         "source": "git@github.com:sifive/example-clic-nested-interrupts.git"
     },
     {
-        "commit": "27a9b3b44b07598e8be6492edb27c09b611eec8f",
+        "commit": "2b5425aac870ac892ed1e08134b00a6e0184a062",
```

Comment:
This update also picked "add check on pmp granularity" changes. I hope you already regress this on all cores

Reply:
exactly,  the example-freertos-pmp-blinky, check granularity add the number of PMP, if it is not compliant with the program we will print a message on the uart and exit with 0.

Reply:
ok. A nop for non-compliance case, which should gracefully exit